src/components/board/board.test.js

```diff
@@ -1,8 +1,36 @@
+const board = require('../../services/boardService');
```

Comment:
This test isn't working - need to use 'import board....' and not the boardService - it doesn't exist.